change/@fluentui-react-motion-components-preview-e172abef-b853-45cb-a027-414535ffd1cd.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32936&commitId=50b86c2059cfbb6780f39cdc8c9504a730941fde&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|3| Changed|
|Avatar Converged.badgeMask.chromium.png|37| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1147| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|984| Changed|
|Drawer.Full Overlay RTL.chromium.png|1166| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>